packages/slice-machine/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -34,8 +42,26 @@ const dataTipText2 = `The repeatable zone is for a group<br/>
 
 const FieldZones: FC = () => {
   const { slice, setSlice, variation } = useSliceState();
+  const [
+    isDeleteRepeatableZoneDialogOpen,
+    setIsDeleteRepeatableZoneDialogOpen,
+  ] = useState(false);
+
+  // We won't show the Repeatable Zone if no items are configured.
+  const hasItems = Boolean(
+    variation.items && Object.keys(variation.items).length > 0,
+  );
 
   const _onDeleteItem = (widgetArea: WidgetsArea) => (key: string) => {
+    if (
```

Comment:
_✍️ potential change_

Here I think we should add a condition with the Amplitude experiment value. It would let us merge branches without having to wait everything. WDYT?

Reply:
Yep, totally forgot about the experiment! I added `useGroupsInSlicesExperiment()` and the correct checks.